features/polyExplorer/src/screens/stories/story-utils.js

```diff
@@ -61,3 +63,95 @@ export function latestActiveUsersValue(product) {
     }
     return Math.floor(latest.user_count / 1000000);
 }
+
```

Comment:
@RichardSto I created this two functions that I need for the Data Types component. Most likely, we will use it only there. Does it make sense that I write the functions here then or is it better if they are in the component?

Reply:
No it's fine here!

Reply:
I would even say some of them would belong in poly-look, `createBubbleArray` for instance.